src/Mvc/Mvc.Core/src/FileStreamResult.cs

```diff
@@ -79,5 +81,37 @@ public override Task ExecuteResultAsync(ActionContext context)
             var executor = context.HttpContext.RequestServices.GetRequiredService<IActionResultExecutor<FileStreamResult>>();
             return executor.ExecuteAsync(context, this);
         }
+
+        async Task IResult.ExecuteAsync(HttpContext httpContext)
+        {
+            var loggerFactory = httpContext.RequestServices.GetRequiredService<ILoggerFactory>();
+            var logger = loggerFactory.CreateLogger<RedirectResult>();
```

Comment:
Is `RedirectResult` the relevant type parameter for this logger?

Reply:
No, it should be `FileStreamResult`, thanks! already fixed